content/ember-released-4-10.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Ember 4.10 Released
+authors:
+  - the-ember-learning-team
+date: 2023-01-13T00:00:00.000Z
+tags:
+  - releases
+  - '2023'
+  - version-4-x
+---
+
+Today the Ember project is releasing version 4.10 of Ember.js, Ember Data, and Ember CLI.
+
+This release kicks off the 4.11 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/releases/)
+- [The Ember Release Cycle](https://blog.emberjs.com/new-ember-release-process/)
+- [The Ember Project](https://blog.emberjs.com/ember-project-at-2-0/)
+- [Ember LTS Releases](https://blog.emberjs.com/announcing-embers-first-lts/)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 4.10
+
+Ember.js 4.10 is an incremental, backwards compatible release of Ember with bug fixes, performance improvements, and minor deprecations.
+
+#### Bug Fixes
+
+Ember.js 4.10 introduced 2 bug fixes.
+
+- [#20327](https://github.com/emberjs/ember.js/pull/20327) - Fix the types for the mutation-methods of `NativeArray`
+- [#20283](https://github.com/emberjs/ember.js/pull/20283) - revert TS `compilerOptions.target` to ES2017
+
+#### Features
+
+Ember.js 4.10 introduced 1 features.
+
+- [#20270](https://github.com/emberjs/ember.js/pull/20270) / [#20271](https://github.com/emberjs/ember.js/pull/20271) - Add new imports for `getOwner` and `setOwner` from `@ember/owner` and introduce new `@ember/routing` sub-modules
+
+#### Deprecations
+
+Ember.js 4.10 introduced 3 deprecations.
+
+- [#20341](https://github.com/emberjs/ember.js/pull/20341) - Deprecations for importing `htmlSafe` and `isHTMLSafe` from `@ember/string`. They have moved to `@ember/template`
+- [#20344](https://github.com/emberjs/ember.js/pull/20344) - Deprecate `@ember/string` when used from ember-source. You should use the `@ember/string`-addon
+- [#20342](https://github.com/emberjs/ember.js/pull/20342) - Deprecate @ember/error
+
+#### Types
+
+Ember.js 4.10 introduced 2 types.
+
+- [#20253](https://github.com/emberjs/ember.js/pull/20253) - Add the `Resolver` type to preview types
+- [#20319](https://github.com/emberjs/ember.js/pull/20319) - resolve services with `Owner.lookup`
```

Comment:
Oh no – we should not break types out into their own section!

1. Types are *features*, and they serve everyone, whether people are directly using TS or not (your editor tooling will take advantage of them)!
2. TypeScript is *an officially supported language* for the project, so even the rare TS features which only affect TS users are *first-class features*, and deserve the same billing as any other.